src/Services/OData/ODataFormatter.cs

```diff
@@ -260,7 +287,19 @@ private void WriteSingleRefContent(object references, PortalContext portalContex
                 }
             }
         }
+        /// <summary>
+        /// Writes the given Content list to the webresponse.
+        /// </summary>
+        /// <param name="portalContext">The current <see cref="PortalContext"/> instance containing the current webresponse.</param>
+        /// <param name="contents"></param>

```

Comment:
some comment is missing for the contents param

Reply:
ok